api/services/elastigroup/azure/scaleSetVms/paths/groups.yaml

```diff
@@ -28,30 +28,7 @@ post:
       application/json:
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master